packages/provider-elasticsearch/src/example-types/metricGroups/pivot.js

```diff
@@ -362,7 +377,10 @@ let createPivotScope = (node, schema, getStats) => {
       groups: rows,
     })
 
-    let statsAggs = { aggs: getAggsForValues(node.values) }
+    let aggValues = _.reject('skip', node.values)
+    let statsAggs = _.isEmpty(aggValues)
+      ? {}
```

Comment:
Do we need special handling for this? We didn't handle this before if node.values was empty

Reply:
We didn't it just produces a strange query with {aggs: {}} nested in it. 
I've added this case handling to produce cleaner queries